connector/src/test/scala/com/basho/riak/spark/rdd/connector/RiakConnectorTests.scala

```diff
@@ -17,12 +31,14 @@ class RiakConnectorTests {
 
   @Test
   def openSessionWithDefaultParametersShouldReturnSessionFromPool(): Unit = {
-    val session_1 = connector_1.openSession()
+    mockSessionCache(2, new RiakClient(mock(classOf[RiakCluster])))
```

Comment:
Since mockSessionCache used in each test case it makes sense to move it in @Before method